tests/0113-cooperative_rebalance.cpp

```diff
@@ -1998,21 +1994,24 @@ static void n_wildcard() {
 
     if (Test::assignment_partition_count(c1, NULL) == 1 &&
         Test::assignment_partition_count(c2, NULL) == 1 && deleted_topic) {
-      /* Callback count can vary in KIP-848 */
+      /* accumulated in lost case as well for the classic protocol*/
+      TEST_ASSERT(rebalance_cb1.revoke_call_cnt == 1,
+                  "Expecting C_1's revoke_call_cnt to be 1 not %d",
+                  rebalance_cb1.revoke_call_cnt);
+      TEST_ASSERT(rebalance_cb2.revoke_call_cnt == 1,
+                  "Expecting C_2's revoke_call_cnt to be 1 not %d",
+                  rebalance_cb2.revoke_call_cnt);
+
+      /* Deleted topics are not counted as lost in KIP-848.
+       * Assignment changes are propogated through ConsumerGroupHeartbeat. */
       if (test_consumer_group_protocol_classic()) {
-        /* accumulated in lost case as well */
-        TEST_ASSERT(rebalance_cb1.revoke_call_cnt == 1,
-                    "Expecting C_1's revoke_call_cnt to be 1 not %d",
-                    rebalance_cb1.revoke_call_cnt);
-        TEST_ASSERT(rebalance_cb2.revoke_call_cnt == 1,
-                    "Expecting C_2's revoke_call_cnt to be 1 not %d",
-                    rebalance_cb2.revoke_call_cnt);
+        expected_lost_cnt++;
       }
 
-      TEST_ASSERT(rebalance_cb1.lost_call_cnt == 1,
+      TEST_ASSERT(rebalance_cb1.lost_call_cnt == expected_lost_cnt,
                   "Expecting C_1's lost_call_cnt to be 1 not %d",
```

Comment:
Here write `expected_lost_cnt`
```suggestion
                  "Expecting C_1's lost_call_cnt to be %d not %d",
```